src/pages/[chapterId]/index.tsx

```diff
@@ -54,20 +58,27 @@ const Chapter: NextPage<ChapterProps> = ({
   );
 };
 
+const AYAH_KURSI_SLUGS = ['ayatul-kursi', 'آیت الکرسی'];
+
 export const getStaticProps: GetStaticProps = async ({ params, locale }) => {
-  const chapterIdOrVerseKeyOrSlug = String(params.chapterId);
+  let chapterIdOrVerseKeyOrSlug = String(params.chapterId);
   let isChapter = isValidChapterId(chapterIdOrVerseKeyOrSlug);
   // initialize the value as if it's chapter
   let chapterId = chapterIdOrVerseKeyOrSlug;
-  // if it's not a valid chapter id and it's not a valid verse key, we reject it.
+  // if it's not a valid chapter id and it's not a valid verse key, will check if it's Ayat Al kursi or if it's a Surah slug
   if (!isChapter && !isValidVerseKey(chapterIdOrVerseKeyOrSlug)) {
-    const sluggedChapterId = await getChapterIdBySlug(chapterIdOrVerseKeyOrSlug, locale);
-    // if it's not a valid slug
-    if (!sluggedChapterId) {
-      return { notFound: true };
+    // if the value is a slug of Ayatul Kursi
+    if (AYAH_KURSI_SLUGS.includes(chapterIdOrVerseKeyOrSlug.toLowerCase())) {
+      chapterIdOrVerseKeyOrSlug = '2:255';
+    } else {
+      const sluggedChapterId = await getChapterIdBySlug(chapterIdOrVerseKeyOrSlug, locale);
+      // if it's not a valid slug
+      if (!sluggedChapterId) {
+        return { notFound: true };
+      }
+      chapterId = sluggedChapterId;
+      isChapter = true;
```

Comment:
nit: the `if`s a bit too nested.

Is it possible to reduce the number of nested `if's ? 

for example
```
if (!isChapter && !isValidVerseKey(chapterIdOrVerseKeyOrSlug)) {
     // if the value is a slug of Ayatul Kursi
     if (AYAH_KURSI_SLUGS.includes(chapterIdOrVerseKeyOrSlug.toLowerCase())) {
       chapterIdOrVerseKeyOrSlug = '2:255';
```

maybe could  be

```
check if the value is not chapter, not a verse key, and not ayatul curse
if (!isChapter && !isValidVerseKey(chapterIdOrVerseKeyOrSlug) && AYAH_KURSI_SLUGS.includes(chapterIdOrVerseKeyOrSlug.toLowerCase())) {
       chapterIdOrVerseKeyOrSlug = '2:255';

```

or the better way, we can turn it into a function so it's easier to understand.